### PR TITLE
moved everthing from other branch to prevent merge history conflict

### PR DESCRIPTION
**Pull request by billycz8**

### CODE REVIEW

**Comment by watwars** [comment]
Why is this PR so huge.

I am not sure how a Java project works but I see there are a ton of empty files. Can those be removed?

Also can some of the files be added to `.gitignore`?

File: frontend/FarmWise/app/src/main/java/com/example/farmwise/FarmFragment.java [hunk 1]
```diff
@@ -1,11 +1,81 @@
+//package com.example.farmwise;
```
Comment (watwars) [inline frontend/FarmWise/app/src/main/java/com/example/farmwise/FarmFragment.java]: this file can probably be removed if everything is commented out

Comment (billycz8) [reply frontend/FarmWise/app/src/main/java/com/example/farmwise/FarmFragment.java]: Working on fixing it

Comment (billycz8) [reply frontend/FarmWise/app/src/main/java/com/example/farmwise/FarmFragment.java]: Removed files from build and changed the commented file, the rest of the files are part of the language model I used and the grammar I created for crops so they are needed.

**Review by watwars — APPROVED** [review]

**Pull request merged**